tools/src/main/scala/scala/scalanative/interflow/Inline.scala

```diff
@@ -20,17 +20,17 @@ trait Inline { self: Interflow =>
       .fold[Boolean] {
         false
       } { defn =>
-        val isCtor = originalName(name) match {
+        lazy val isCtor = originalName(name) match {
```

Comment:
This and following `lazy val`s can be potentially replaced with `def`s. They're used only once and `lazy val`s implies some amounts of overhead.

Reply:
Changed in a1df6de.